stories/object-number/object-number.stories.js

```diff
@@ -0,0 +1,79 @@
+import '../../dist/object-number.css';
+import './custom.scss';
+
+export default {
+    title: 'Components/Object Number',
+    parameters: {
+        description: `The Object Number displays number and number unit properties for an object.
```

Comment:
@jessmetivier should this description be in the documentation page instead of mine `The Object Number displays number and number unit properties for an object.......`?

Reply:
@droshev @InnaAtanasova @JKMarkowski @prsdthkr how do you think, how I could show semantic and a11y aspects in a correct and understandable way? Maybe by adding a separate demo section with an example as @prsdthkr add [here](https://github.com/SAP/fundamental-styles/pull/1419#discussion_r474139978) _(object number with an icon and description text)_?

Reply:
Hi @olshansky, I'm not sure if I understand your question... are you asking me whether the description I wrote should replace the one that's already there?

Reply:
@olshansky - maybe we can remove the accessibility tag for now and open an issue (for future) to create an example that demonstrate the a11y part.

Reply:
@jessmetivier yes, exactly about that

Reply:
@droshev ok, thanks. I've removed a11y tag already. Will create an issue

Reply:
@olshansky If that's the case, then yes. This description should replace it.